### PR TITLE
feat: Expose to Attribute related panelID

### PR DESCRIPTION
**Pull request by M123-dev**
### What
- Exposing to Attribute related panelID

### Part of 
- https://github.com/openfoodfacts/smooth-app/issues/1029

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,6 +24,9 @@ ios/
 *.ipr
 .idea/
 
+# VSCode
+.vscode/
+
```
Comment (monsieurtanuki) [inline .gitignore:29]: ???

File: lib/model/Attribute.dart [hunk 2]
```diff
@@ -107,6 +111,7 @@ class Attribute extends JsonObject {
   final String? descriptionShort;
   final double? match;
   final String? status;
+  final String? panelID;
```
Comment (monsieurtanuki) [inline lib/model/Attribute.dart:114]: I would say `panelId` instead.

**Review by monsieurtanuki — APPROVED** [review]
@M123-dev Looks good to me! I have minor comments: read them but feel free to ignore them.

File: lib/model/Attribute.dart (hunk 2 above)
Comment (M123-dev) [reply lib/model/Attribute.dart]: 👍🏼

File: .gitignore (hunk 1 above)
Comment (M123-dev) [reply .gitignore:29]: Tried dart in vscode today and yeah thils folder was missing in the gitignore

**Pull request merged**